source/Sashimi/Sashimi.csproj

```diff
@@ -23,10 +23,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Octopus.Configuration" Version="4.0.0" />
```

Comment:
actually, should we update this to 4.1.0. As we have updated it in this PR https://github.com/OctopusDeploy/Sashimi.AzureWebApp/pull/25/files

Reply:
In that PR, the bump was from `3.0.0` to `4.1.0`, necessitated by the `net5.0` upgrade (v3 of the Octopus.Configuration library didn't have an appropriate build target to use in a `net5.0` project... I think v3 only supported `netfx`).

I'm generally being as conservative as possible on the surface area of changes and version bumps... with exception, I guess, of the Server.Extensibility ones, which had functional changes and was required to get tests passing.

Reply:
Ok sounds reasonable.